addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -245,8 +244,16 @@ const PopupWidget = publicWidget.Widget.extend({
     /**
      * @private
      */
-    _onHashChange() {
-        this._showPopupOnClick();
+    _onHashChange(ev) {
+        if (ev && ev.newURL) {
+            // Keep the new hash from the event to avoid conflict with the eCommerce
+            // hash attributes managing.
+            // TODO : it should not have been a hash at all for ecommerce, but a
+            // query string parameter
+            this._showPopupOnClick(new URL(ev.newURL).hash);
+        } else {
+            this._showPopupOnClick();
+        }
```

Comment:
(ev.newURL is probably useless in the check but it's fine)
In master we can just take `ev` for granted and remove the safety net.

Reply:
Ok, I'll do the necessary job in master.